cluster/saltbase/salt/kubelet/default

```diff
@@ -138,9 +138,9 @@
 {% if pillar.get('network_provider', '').lower() == 'opencontrail' %}
   {% set network_plugin = "--network-plugin=opencontrail" %}
 {% elif pillar.get('network_provider', '').lower() == 'cni' %}
-  {% set network_plugin = "--network-plugin=cni --network-plugin-dir=/etc/cni/net.d/" %}
+  {% set network_plugin = "--network-plugin=cni --network-plugin-dir=/etc/cni/net.d/ --cni-bin-dir=/opt/cni/bin" %}
 {%elif pillar.get('network_policy_provider', '').lower() == 'calico' and grains['roles'][0] != 'kubernetes-master' -%}
-  {% set network_plugin = "--network-plugin=cni --network-plugin-dir=/etc/cni/net.d/" %}
+  {% set network_plugin = "--network-plugin=cni --network-plugin-dir=/etc/cni/net.d/ --cni-bin-dir=/opt/cni/bin" %}
```

Comment:
That would be one way to do it.  I didn't think to change where the CNI binaries were installed for debian to make them consistent.  Why not just use `/home/kubernetes/cni` for all OSes?  If that's what you're asking in the linked issue, I'd much rather that be the case than having N different potential locations.